core/trino-main/src/main/java/io/trino/execution/RenameViewTask.java

```diff
@@ -88,6 +89,12 @@ public ListenableFuture<Void> execute(
         if (metadata.isView(session, target)) {
             throw semanticException(TABLE_ALREADY_EXISTS, statement, "Target view '%s' already exists", target);
```

Comment:
Unrelated to your PR: Can you please add a new commit for changing the error code from `TABLE_ALREADY_EXISTS` to `GENERIC_USER_ERROR` in this specific case ? (pls mind also the corresponding test case)

Reply:
Done! also add the UT for this. Please help review again, Thanks!